content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,11 +99,12 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
 Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
 supports Linux as well as Windows.
```

Comment:
"supports both Windows and Linux", the current wording is odd if you're no longer contrasting it with a Windows-only container image.